src/neuralnet/superblock.h

```diff
@@ -236,6 +236,15 @@ class Superblock
     //!
     static constexpr uint32_t CURRENT_VERSION = 2;
 
+    //!
+    //! \brief The maximum allowed size of a serialized superblock in bytes.
+    //!
+    //! The bulk of the superblock data is comprised of pairs of CPIDs to
+    //! magnitude values. A value of 4 MB provides space for roughly 200k
+    //! CPIDs that accumulated a non-zero magnitude.
+    //!
+    static constexpr size_t MAX_SIZE = 4 * 1000 * 1000;
```

Comment:
Should we use MiB instead of MB here? i.e. 4 * 1024 * 1024? Don't really have a strong opinion but wanted to note it.

Reply:
The block size constants are defined in MB, so I just followed that convention. Besides that, I don't have a preference either.

Reply:
I'm good with it as it is.